prql-elixir/native/prql/src/lib.rs

```diff
@@ -0,0 +1,161 @@
+use rustler::{Atom, NifResult, NifStruct, NifTuple};
+
+mod atoms {
+    rustler::atoms! {
+      ok,
+      error,
+
+      // dialects
+      ansi,
+      big_query,
+      click_house,
+      generic,
+      hive,
+      mssql,
+      mysql,
+      postgres,
+      sql_lite,
+      snow_flake
```

Comment:
Nit: these are generally one word, so wouldn't have a `_` separator

(though I guess `ClickHouse` has two upper case characters, but `Snowflake` doesn't. Regardless most consistent would be to just use a single word)

Reply:
is `BigQuery` from google also considered as a single word?

Reply:
Ha, I think it's reasonable to have them all as a single word. If I look at examples like https://pypi.org/project/google-cloud-bigquery/ or https://clickhouse.com/docs/en/integrations/language-clients/python/intro/, they're all a single word....

Reply:
Shall i also use a single word for `ClickHouse`? `clickhouse` would be elixir representation

Reply:
Yes, I think for all of them

Reply:
Awesome, on it

Reply:
Fixed